### ISSUE TITLE
Change slide dragging less than half page

### ISSUE THREAD

**Issue by luigicarlini1988** [skidding/dragdealer#77]
Hi guys, i noticed that creating a slideshow with dragdealer, like in this demo:

http://tympanus.net/Development/DraggableDualViewSlideshow/

For passing from one slide to another, the script needs the user to drag at least 51% of the .current slide. Is there any way to decrease that value or make the "drag-change slide" more sensible?
Like, i drag and scroll only 100px of my handle and i pass to the other slide?

**Comment by Sedmica** [comment]
I have this problem too!

**Comment by skidding** [comment]
You can make the slide change without sliding >50% if you release while moving at speed. It will be easier to achieve this effect if you increase [the speed option.](https://github.com/skidding/dragdealer/blob/master/src/dragdealer.js#L87-L89)

Let me know if this works.

**Comment by antonellopasella** [comment]
Yeah, it works, but will be better ho have a "threshold" option (in px or %) to trigger "nextslide" when drag reaches this amount

**Comment by rajala** [comment]
Have anyone been able to figure this one out? I've been looking at the code for a while but can't figure out how to change this. I should probably learn js.

**Comment by christru** [comment]
Anyone come up with a work around for implementing threshold?

**Comment by christru** [comment]
fyi for what its worth this fixed my issue. Setting inertia to 15
https://github.com/kaigth/dragdealer/blob/add-features/src/dragdealer.js

**Comment by amhoho** [comment]
I have this problem too! @christru it's not working!
